critiquebrainz/frontend/templates/artist/entity.html

```diff
@@ -48,7 +48,17 @@
 
 <div class="row">
   <div class="col-sm-9">
-    <h4>{{ _('Discography') }}</h4>
+    <div class="row">
+      <div class="col-sm-8">
+        <h4>{{ _('Discography') }}</h4>
+      </div>
+      <div class="col-sm-4">
+        <div>
+          <input type="checkbox" name="only-reviewed" value="1" class="only-reviewed" onchange="showOnlyReviewed()"/>
```

Comment:
instead of using an html attribute here, other places in the CB code us jquery to install an event handler: https://github.com/metabrainz/critiquebrainz/blob/483594644b4954c3a62f89c84d9abf1e97b14a8a/critiquebrainz/frontend/templates/review/modify/base.html#L76
We should do the same thing for consistency